backend/.gitignore

```diff
@@ -1,4 +1,6 @@
 firebase-debug.log
+service-account.json
```

Comment:
We should document in the readme that this is required in order to deploy the Cloud Functions backend.

Reply:
oh right, I meant to do that but got sidetracked. But FWIW this is not actually required for deployment; it was only used for local testing (it let me write to Firestore from my local env)